browser/src/control/Control.NavigatorPanel.ts

```diff
@@ -209,6 +209,8 @@ class NavigatorPanel extends SidebarBase {
 			this.showNavigationPanel();
 			$('#navigator-dock-wrapper').show(200);
 			app.map.uiManager.setDocTypePref('ShowNavigator', true);
+			// this will update the indentation marks for elements like ruler
+			app.map.fire('moveend');
```

Comment:
hmm, can we use some dedicated signal?
I think this one is called when we move the view. It might have some other implications (for movement animation, zooming etc).
Better to avoid unnecessary side effects :)